### ISSUE TITLE
Maximum number of field columns

### ISSUE THREAD

**Issue by argenturatbek** [tomansley/SimpliListViews#141]
**Is your feature request related to a problem? Please describe.**
To accommodate a large number of columns, consider setting a maximum limit on the number of fields displayed. Additionally, it may be helpful to set an initial width for each column and provide horizontal scrolling functionality to allow users to navigate through the data easily.

**Describe the solution you'd like**
How many field columns can be added at maximum in a list view?

**Describe alternatives you've considered**
So far, I have attempted to add 33 fields.

**Additional context**

**Comment by tomansley** [comment]
Hello Argen,

Great to hear from you.

I am a little confused as to what the feature request is asking for. Are
you asking for a setting to be able to set a limit on the number of fields
available to be displayed in a list view?

Are you also asking for a setting that could be used as the initial width
of the columns?

I will look into allowing a horizontal scrolling capability as well.

Let me know if what I say above is correct.

Cheers
Tom

**Comment by argenturatbek** [comment]
Hello Tom, 

1. Yes, you are right. Our business requirement is to put more than 100 fields on the list view. Thus, I was asking whether how many fields we can put at max?

2. I've tried setting the initial width for the columns so that it looks normal. But the initialWidth is not being reflected. 

Thank you.

**Comment by watsonps** [comment]
I am also experiencing some issues with width of columns. Columns that don't fit are challenging to impossible to resize since there is no horizontal scrolling.

**Comment by tomansley** [comment]
@watsonps We now allow horizontal scrolling in the latest release. Please let me know if this does not resolve your issue.

**Comment by watsonps** [comment]
> [@watsonps](https://github.com/watsonps) We now allow horizontal scrolling in the latest release. Please let me know if this does not resolve your issue.

Yes! Horizontal scrolling did resolve this issue, thank you for your work on this! I am looking forward to a fix for #158 which will allow us to move to the latest release
